package.json

```diff
@@ -32,38 +32,21 @@
   "homepage": "https://github.com/digitalbazaar/bedrock-quasar",
   "dependencies": {
     "animate.css": "^4.1.1",
-    "quasar": "^2.7.3"
+    "quasar": "^2.14.6"
   },
   "peerDependencies": {
-    "@bedrock/vue": "^5.0.0",
-    "vue": "^3.2.36"
+    "@bedrock/vue": "^5.0.1",
+    "vue": "^3.3.4"
   },
   "devDependencies": {
-    "eslint": "^8.18.0",
-    "eslint-config-digitalbazaar": "^4.0.1",
-    "eslint-plugin-jsdoc": "^39.3.3",
-    "eslint-plugin-unicorn": "^42.0.0",
-    "eslint-plugin-vue": "^9.1.1"
-  },
-  "bedrock": {
-    "webpack": {
-      "animate.css": {
-        "resolve": {
-          "alias": {
-            "animate.css": "animate.compat.css"
-          }
-        }
-      },
-      "quasar": {
-        "resolve": {
-          "alias": {
-            "quasar$": "dist/quasar.umd.js"
-          }
-        }
-      }
-    }
+    "eslint": "^8.57.0",
+    "eslint-config-digitalbazaar": "^5.0.1",
+    "eslint-plugin-jsdoc": "^48.2.0",
+    "eslint-plugin-quasar": "^1.1.0",
+    "eslint-plugin-unicorn": "^51.0.1",
+    "eslint-plugin-vue": "^9.22.0"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=18"
```

Comment:
I know this does not inform a major release, but does it at least get a mention in the changelog?

Reply:
It can be mentioned.  What should it say?  There are no direct tests here and I'm not sure what the actual requirement is.  This is mostly to lock to modern runtimes.

Reply:
Required Node.js 18+.